Okta.Sdk.Abstractions.UnitTests/UserAgentHelperShould.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using FluentAssertions;
+using Xunit;
+
+namespace Okta.Sdk.Abstractions.UnitTests
+{
+    public class UserAgentHelperShould
+    {
+        [Theory]
+        [InlineData("foo")]
+        [InlineData("foo .NET")]
+        [InlineData(".NET")]
+        [InlineData(".NET foo")]
+        [InlineData(".NET Framework 4.8")]
+        public void ReturnPassedFrameworkDescriptionIfItDoesNotStartWithKeyword(string runtimeDescription)
+        {
+            // Keyword for runtimeDescription = ".NET Core"
+            var frameworkInfo = UserAgentHelper.GetRuntimeVersion(runtimeDescription);
+
+            frameworkInfo.Should().Be(runtimeDescription);
+        }
+
+        [Theory]
+        [InlineData("file:///C:/Program Files/dotnet/shared/Microsoft.NETCore.App/2.0.9/System.Private.CoreLib.dll", "2.0.9")]
+        [InlineData("Microsoft.NETCore.App/2.0.9/System.Private.CoreLib.dll", "2.0.9")]
+        [InlineData("foo/Microsoft.NETCore.App/2.0.9/System.Private.CoreLib.dll", "2.0.9")]
+        [InlineData("foo/Microsoft.NETCore.App/3.1.1/System.Private.CoreLib.dll", "3.1.1")]
```

Comment:
Might be useful to also have paths that include backslashes as well, since Windows will often report paths in that format.

Reply:
Since we are using `GetTypeInfo().Assembly?.CodeBase` to get the `runtimeAssemblyCodeBasePath`  it is not necessary because it always uses `/`.